### PR TITLE
Add column to show promotion status in dashboard

### PR DESCRIPTION
**Pull request by oleg-nenashev**
It's a reworked version of https://github.com/jenkinsci/promoted-builds-plugin/pull/30

@reviewbybees @jglick

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
#60 is an alternative version

**Comment by AnneTheAgile** [comment]
This got approval on #30 to be merged. Thank you @oleg-nenashev and to Cloudbees!

**Comment by oleg-nenashev** [comment]
I'm going to merge both this PR and #60. This one will be renamed to the "Last build promotions"

**Comment by oleg-nenashev** [comment]
@reviewbybees @AnneTheAgile 
New version is ready for the review

File: src/main/java/hudson/plugins/promoted_builds/LastBuildPromotionStatusColumn.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            final Run<?, ?> b = job.getLastBuild();
+            PromotedBuildAction a = b != null ? b.getAction(PromotedBuildAction.class) : null;
+            if (a != null) {
+                for (Status s : a.getPromotions()) {
+                    icons.add(s.getIcon("16x16"));
+                }
+            }
+        }
+        return icons;
+    }
```
Comment (tfennelly) [inline src/main/java/hudson/plugins/promoted_builds/LastBuildPromotionStatusColumn.java]: NIT: delay creating of `icons` until needed, returning an `emptyList` if not needed.

**Comment by tfennelly** [comment]
:bee: 

Not worth adding a unit test for it?

**Comment by oleg-nenashev** [comment]
@tfennelly 
Added a unit test

**Comment by tfennelly** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
@reviewbybees done

Merging as is, because I need to retain the authorship of the first commit.

**Pull request merged**